zkevm-circuits/src/state_circuit/constraint_builder.rs

```diff
@@ -273,6 +273,8 @@ impl<F: Field> ConstraintBuilder<F> {
             self.require_zero("memory address fits into 2 limbs", limb.clone());
         }
 
+        // TODO: check address is aligned (addr%32==0)
```

Comment:
would you plan to implement the todo in this PR? seems not high priority.

Reply:
No, that can be done separately.

Reply:
that is OK!